calendar/index.js

```diff
@@ -157,6 +157,12 @@ class IRNMNCalendar extends HTMLElement {
         this.verifyOpenDate();
         this.render();  
         this.loadFromSessionStorage();
+
+        Promise.resolve().then(() => {
```

Comment:
@samuelsilvapt I don't really understand what is happening here. You are creating a promise but you are not waiting for anything that is running async..? or am I misunderstanding what is happening

Reply:
@ejcromwell  an empty promise (with resolve() ) runs after the DOM updates

Reply:
I see. Would MutationObserver() be a better option. Not fussed either way, but if just using a promise, lets leave a comment to explain what DOM changes we are wairing for?

Reply:
I added a comment to the promise. MutationObserver() looks useful for cases like this, I wasn't familiar with

Reply:
comment works.